cmd/util/ledger/migrations/fix_broken_data_migration_test.go

```diff
@@ -119,7 +119,7 @@ func TestFixSlabsWithBrokenReferences(t *testing.T) {
 	accountStatusRegisterID := mustDecodeHex("612e73")
 	updatedAccountStatusRegister := ledger.NewPayload(
 		ledger.NewKey([]ledger.KeyPart{ownerKey, {Type: 2, Value: accountStatusRegisterID}}),
-		ledger.Value([]byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7, 0xbe, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x9, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}),
+		[]byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7, 0xba, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x9, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0},
```

Comment:
@fxamacker Not sure why this changed. Is that expected?

Reply:
@turbolent This appears to be related to changes in account status register format.  

New account status register size is 4 bytes less than old format because public key count size is changed from 8 bytes to 4 bytes.

https://github.com/onflow/flow-go/blob/dcc3556c21f0fc8872d1db6c5eef9fa870e0a30d/fvm/environment/accounts_status.go#L33-L43

Reply:
Ah, right! Thanks for the explanation and confirming this is correct 🙏